Cargo.toml

```diff
@@ -28,7 +28,7 @@ ic-ethereum-types = { git = "https://github.com/dfinity/ic", rev = "release-2024
 ic-nervous-system-common = { workspace = true }
 ic-metrics-encoder = { workspace = true }
 ic-stable-structures = { workspace = true }
-ic-canister-log = { git = "https://github.com/dfinity/ic", rev = "b7b0e54832bbe6bf4b80a2f75f4252699f5878e0", package = "ic-canister-log" }
+ic-canister-log = { workspace = true }
```

Comment:
@gregorydemay, this seems to work without any issues now that we've removed the `cketh-minter` dependency.